docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,199 @@
+
+In general, if the issue description and title are perceived as questions no more information is needed.
+
+### Good practices
+
+To make issue handling easier for everyone, it is suggested to:
+
+- Make sure that issue titles are named to explain the subject of the issue, have correct spelling, and don't include irrelevant information and/or sensitive information.
+- Make sure that issue descriptions don't include irrelevant information.
+- Make sure that issues do not contain sensitive information.
+- Make sure that issues have all relevant fields filled in.
+-  If an issue is unclear, then try to edit the title and description for more clarity or leave a comment requesting that edits to the issue be made.
+
+> **Note:** Above rules are applicable to both new and existing issues.
+
+### Dealing with missing information
+
+Depending on the issue, you might not feel all this information is needed. Use your best judgement. If you cannot triage an issue using what its author provided, explain kindly to the author that they must provide the above information to clarify the problem. Label issue with `triage/needs-information`.
+
+If the author provides the standard information but you are still unable to triage the issue, then request additional information. Do this kindly and politely because you are asking for more of the author's time. Label issue with `triage/needs-information`. `triage/needs-information`.
+
+If the author does not respond to the requested information within the timespan of a week, close the issue with a kind note stating that the author can request for the issue to be reopened when the necessary information is provided.
+
+If you receive a notification that additional information was provided but you are no longer working on issue triage, then you should delegate the issue to the current person working on issue triage.
+
+## 3. Categorizing an issue
+
+### Duplicate issues
+
+Make sure it's not a duplicate by searching existing issues using related terms from the issue title and description. If you think the issue may be a potential duplicate and can't find the original existing issue, then please reach out to one of the maintainers and ask for help. If you identify that the issue is a duplicate of an existing issue:
+
+1. Add a comment `duplicate of #<issue number>`
+2. Add the `triage/duplicate` label
+
+### Bug reports
+
+If it's not perfectly clear that it's an actual bug, quickly try to reproduce it.
+
+**It's a bug/it can be reproduced:**
+
+1. Add a comment describing detailed steps for how to reproduce it, if applicable.
+2. If maintainers aren't able to address the issue in a timely manner, then label the issue with `help wanted` and an optional `beginner friendly`. Also, include pointers to the problematic source code. Doing this allows us to solicit help from the community to get the issue resolved.
+3. Move on to [prioritizing the issue](#4-prioritization-of-issues).
+
+**It can't be reproduced:**
+
```

Comment:
This isn't grammatical and is a little hard to read. (Having two successive "Either" statements is confusing.)

Please delete the two "Either"s at the beginning of these items and do one of the following:

- Precede the two items with "Either", then add an "or" between the two items.
- Precede these two list items with "Do one of the following:" (This implies the Either/or.)

Reply:
Addressed as suggested

Reply:
There are still two preceding "Either"s. Please delete the one for alternative "2".